package.json

```diff
@@ -51,6 +51,8 @@
   },
   "dependencies": {
     "@sentry/node": "^6.16.1",
-    "dotenv": "^10.0.0"
+    "dotenv": "^10.0.0",
+    "logform": "^2.3.2",
```

Comment:
Wait, why didn't I explicitly depend on logform in upload-service? This is a good change - nice catch, @slifty!

Reply:
@jasonaowen I was wondering that too but not enough to raise an issue at that point 😂